dashboard/app/models/script_level.rb

```diff
@@ -136,12 +136,8 @@ def end_of_script?
   end
 
   def long_assessment?
-    if assessment
-      if level.properties["pages"] && level.properties["pages"].length > 1
-        return true
-      end
-    end
-    false
+    return false unless assessment
```

Comment:
Though I'm happy to condense to a single line, I think two is more readable, as it fits more with my mental model of the requirements. As evidence, note that the method started with nested ifs.